kolibri/core/notifications/api.py

```diff
@@ -285,6 +300,16 @@ def num_correct(examlog):
     )
 
 
+def num_answered(exam_log):
+    return (
+        exam_log.attemptlogs.values_list("item")
+        .order_by("completion_timestamp")
+        .distinct()
+        .aggregate(complete__sum=Count(Case(When(complete=True, then=1), default=0)))
```

Comment:
Do we need the `Count(Case(When` here instead of just `Sum(`? Does it not sum over booleans properly? (I can imagine that possibly this might break on postgres, but I think it would work on sqlite at least).

Reply:
I initially tried just using `Sum(` but it didn't work, unfortunately. I just went through and tested again to be sure and for whatever reason, I'm not getting the proper updated data as expected when a learner submits an answer to a question.